pkg/web/templates/components/nav.html

```diff
@@ -1,12 +1,12 @@
 {{ define "nav" }}
-<nav class="navbar bg-[#517994] py-4">
+<nav class="navbar shadow-md py-4">
   <div class="navbar-start">
     <div class="dropdown">
       <button type="button" class="ml-2 focus:outline-none focus:ring-2 focus:ring-gray-200 lg:hidden">
-        <i class="fa-solid fa-bars text-lg text-black"></i>
+        <i class="fa-solid fa-bars text-lg"></i>
         <span class="sr-only">Open Navigation Menu</span>
       </button>
-      <ul tabindex="0" class="menu menu-sm dropdown-content mt-3 z-[1] p-2 shadow bg-[#517994] rounded-box w-52 text-white">
+      <ul tabindex="0" class="menu menu-sm dropdown-content bg-gray-300 mt-3 z-[1] p-2 shadow rounded-box w-52 font-semibold">
```

Comment:
A bg color was added for the dropdown menus to ensure visibility as the white on white backgrounds was causing a transparent look. 

I attempted a black bg color and slightly darker grays, but users wouldn't be able to see the active state when hovering over text in the menu. I attempted to adjust the hover state, but there's a style in daisyui that I couldn't override. I can always try again if its decided that a darker color is preferred.